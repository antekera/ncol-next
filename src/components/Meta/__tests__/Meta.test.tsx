import React from 'react'

import { render } from '@testing-library/react'

import { Meta } from '..'

describe('Meta', () => {
  test('should be defined', () => {
    const { container } = render(<Meta />)
    expect(container.firstChild).toBeDefined()
  })
})
