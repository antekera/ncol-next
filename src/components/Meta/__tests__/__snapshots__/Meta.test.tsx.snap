// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meta should match snapshots 1`] = `
<meta
  charset="utf-8"
/>
`;
